### PR TITLE
Add Blog 3 & 4 Descriptions

### PR DESCRIPTION
**Pull request by MarvinBertin**

### CODE REVIEW

File: content/blog/series_1_blog_3/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Ethereum: 2nd Generation Crypto, Web 3.0"
-description: ""
+description: "Introduce Ethereum as the second generation crypto that brought smart contracts to the world. Describe why this new technology is so much more than a “digital currency”, and how it gave rise to the Defi boom and the birth of a new internet."
```
Comment (ggydush) [inline content/blog/series_1_blog_3/index.md:3]: nit: DeFi

File: content/blog/series_1_blog_4/index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Cardano: 3rd Generation Crypto, A World Financial Computer"
-description: ""
+description: "Introduce Cardano as the only true third generation crypto with the potential to overtake the market leaders. Explain why Cardano’s approach to design and engineering sets it apart from other competitors, and how its purpose-driven philosophy, will revolutionize the global financial system by banking the unbanked."
```
Comment (ggydush) [inline content/blog/series_1_blog_4/index.md:3]: nit: third-generation

Comment (ggydush) [reply content/blog/series_1_blog_4/index.md:3]: remove comma after `philosophy`

**Review by ggydush — APPROVED** [review]
LGTM!

File: content/blog/series_1_blog_3/index.md (hunk 1 above)
Comment (Vardominator) [reply content/blog/series_1_blog_3/index.md:3]: You should probably keep the tone the same: "Introduction to...", "Description of..."

**Pull request merged**